apps/evm/test/evm_test.exs

```diff
@@ -301,4 +246,10 @@ defmodule EvmTest do
       {address, storage}
     end)
   end
+
+  def logs_hash(logs) do
+    logs
+    |> ExRLP.encode()
+    |> :keccakf1600.sha3_256()
```

Comment:
do we have access to `exeth_crypto` here?

Reply:
@masonforest no, in this project we do not  have access to `exeth_crypto`. I'll try to replace all direct usages of `:keccakf1600` to `exeth_crypto`  now